recipe/meta.yaml

```diff
@@ -10,13 +10,12 @@ source:
   sha256: a737521126efc3295b818acc951a745d7b914446836c7931b0b52998f3934e2d
 
 build:
-  number: 0
-  skip: true  # [win]
+  number: 1
 
 requirements:
   build:
-    - {{ compiler('fortran') }}
-    - {{ compiler('m2w64_fortran') }}  # [win]
+    - {{ compiler('fortran') }} # [not win]
+    - conda-forge/label/m2w64-experimental::mingw-w64-ucrt-x86_64-gcc-fortran # [win]
```

Comment:
Try restoring these to their original values, as it worked before (it was switched off with 316972ec23fa23ac826f9b2c33e3d37234a3e323). But I do see your attempt to use a more modern Fortran compiler for Windows (newer than [`m2w64-gcc-fortran` 5.3](https://anaconda.org/conda-forge/m2w64-gcc-fortran)).

Reply:
I'll give it a try